core/frontend/src/tile/map/MapLayerFormatRegistry.ts

```diff
@@ -28,42 +28,35 @@ export class MapLayerFormat {
   /** Register the current format in the [[MapLayerFormatRegistry]]. */
   public static register() { IModelApp.mapLayerFormatRegistry.register(this); }
 
-  /**
-   * Allow a source of a specific format to be validated before being attached as a map-layer.
-   * @param _url The URL of the source.
-   * @param _userName The username to access the source if needed.
-   * @param _password The password to access the source if needed.
-   * @param _ignoreCache Flag to skip cache lookup (i.e. force a new server request).
-   * @returns Validation Status. If successful, a list of available sub-layers may also be returned.
-   */
-  public static async validateSource(_url: string, _userName?: string, _password?: string, _ignoreCache?: boolean): Promise<MapLayerSourceValidation> { return { status: MapLayerSourceStatus.Valid }; }
+  /** @deprecated in 4.2. Use the overload that takes a [[MapL]]. */
+  public static async validateSource(_url: string, _userName?: string, _password?: string, _ignoreCache?: boolean, _accesKey?: MapLayerKey): Promise<MapLayerSourceValidation> { return { status: MapLayerSourceStatus.Valid }; }
 
-  /**
-   * Create a [[MapLayerImageryProvider]] that will be used to feed data in a map-layer tile tree.
-   * @param _settings The map layer settings to be applied to the imagery provider.
-   * @returns Returns the new imagery provider.
-   * @beta
-   */
-  public static createImageryProvider(_settings: MapLayerSettings): MapLayerImageryProvider | undefined {
-    assert(false, "Subclasses must override this method.");
-    return undefined;
-  }
+  /** Allow a source object to be validated before being attached as a map-layer.
+    * @beta
```

Comment:
You have prescribed this as the replacement for the deprecated, `@public` method `validateSource`, but this replacement is `@beta`. (Why is everything you've added marked beta?)

Reply:
I wanted to get it adopted in the UI before freezing the API.

Reply:
Ok, but we can't deprecate a public API if the only replacement is a beta (unstable) API. That will have to be done when you promote the replacement to public.